core/src/test/scala/akka/kafka/scaladsl/IntegrationSpec.scala

```diff
@@ -418,5 +420,109 @@ class IntegrationSpec extends TestKit(ActorSystem("IntegrationSpec"))
 
       }
     }
+
+    "successful consume transform produce transaction" in {
```

Comment:
`consume-transform-produce` ?

Reply:
Yes.  That's how it's referred to in KIP-98.  I'll update all references +1